tutorials/cifar10-distributed.ipynb

```diff
@@ -0,0 +1,1123 @@
+            "\t{'batch_size': 1024, 'num_workers': 2, 'shuffle': False, 'pin_memory': True}\n",
+            "2021-09-03 07:27:41,494 CIFAR10-Training INFO: Engine run starting with max_epochs=5.\n",
+            "/usr/local/lib/python3.7/dist-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /pytorch/c10/core/TensorImpl.h:1156.)\n",
+            "  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)\n",
+            "2021-09-03 07:29:42,929 CIFAR10-Training INFO: Epoch[1] Complete. Time taken: 00:02:01\n",
+            "2021-09-03 07:31:44,130 CIFAR10-Training INFO: Epoch[2] Complete. Time taken: 00:02:01\n",
+            "2021-09-03 07:34:05,515 CIFAR10-Training INFO: \n",
+            "Epoch 3 - Evaluation time (seconds): 19.90 - train metrics:\n",
+            " \tAccuracy: 0.5890786082474226\n",
+            "\tLoss: 1.1348281545737355\n",
+            "2021-09-03 07:34:08,479 CIFAR10-Training INFO: \n",
+            "Epoch 3 - Evaluation time (seconds): 2.92 - val metrics:\n",
+            " \tAccuracy: 0.5996\n",
+            "\tLoss: 1.1120291015625\n",
+            "2021-09-03 07:34:08,481 CIFAR10-Training INFO: Epoch[3] Complete. Time taken: 00:02:24\n",
+            "2021-09-03 07:36:09,371 CIFAR10-Training INFO: Epoch[4] Complete. Time taken: 00:02:01\n",
+            "2021-09-03 07:38:10,543 CIFAR10-Training INFO: Epoch[5] Complete. Time taken: 00:02:01\n",
+            "2021-09-03 07:38:30,605 CIFAR10-Training INFO: \n",
+            "Epoch 5 - Evaluation time (seconds): 20.02 - train metrics:\n",
+            " \tAccuracy: 0.6951312822164949\n",
+            "\tLoss: 0.8538916676314836\n",
+            "2021-09-03 07:38:33,587 CIFAR10-Training INFO: \n",
+            "Epoch 5 - Evaluation time (seconds): 2.94 - val metrics:\n",
+            " \tAccuracy: 0.703\n",
+            "\tLoss: 0.8472966796875\n",
+            "2021-09-03 07:38:33,590 CIFAR10-Training INFO: Engine run complete. Time taken: 00:10:52\n",
+            "2021-09-03 07:38:33,643 ignite.distributed.launcher.Parallel INFO: End of run\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "g6s0Rn_mdewW"
+      },
+      "source": [
+        "### TPUs on Colab\n",
+        "\n",
+        "Go to Runtime > Change runtime type and select Hardware accelerator = TPU."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "z6e2NW3kofu8"
+      },
+      "source": [
+        "import os\n",
+        "os.environ[\"CUDA_VISIBLE_DEVICES\"] = \"\"\n",
+        "\n",
+        "spawn_kwargs[\"nproc_per_node\"] = 4\n",
+        "spawn_kwargs[\"start_method\"] = \"fork\"\n",
```

Comment:
I think we can remove that for Colab. This should be automatically set up. Please, confirm that